requirements/constraints.txt

```diff
@@ -102,3 +102,6 @@ maxminddb<2.0.0         # maxminddb 2.0.0 has dropped support for Python 3.5
 path<13.2.0             # path 13.2.0 drops support for Python 3.5
 zipp==1.0.0             # zipp 2.0.0 requires Python >= 3.6
 geoip2<4.0.1            # geoip2 requires Python 3.6
+
+# There are bugs in this version of lti-consumer-xblock that needs to be resolved before we can update.
```

Comment:
If you want to be more accurate: 2.11.0 requires symmetric changes in edx-platform before it will work.